packages/main/src/utils/Strings.js

```diff
@@ -0,0 +1,66 @@
+/**
+	* Escape a regular expression text so that it can be used in a regular expression.
+	* @param {string} text The raw text
+	* @returns The regular expression text, with special characters having been escaped.
+	*/
+export function escapeRegex(text) {
+	return text.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+}
+
+/**
+	* Replace all instances of a particular find text.˝
+	* @param {string} text The text to find/replace on
+	* @param {string} find The string to find
+	* @param {string|(s:string)=>string} replace The replace string or function
```

Comment:
Unfortunately our jsdoc parser can not understand that, can we change it to `{string|function}`?